website/docs/components/rich-tooltip/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,215 @@
+
+![Example of collision detection in the Rich Tooltip](/assets/components/rich-tooltip/rich-tooltip-collision-detection.png)
+
+## Offset
+
+The default recommended distance between the toggle and the Rich Tooltip pointer is 4px.
+
+![Default spacing for the Rich Tooltip](/assets/components/rich-tooltip/rich-tooltip-offset-spacing.png)
+
+!!! Info
+
+In some cases, adding an offset may be necessary to adjust the position of the RichTooltip. Changing the default offset should be done sparingly and only when it’s necessary to make sure that the component does not obscure or cover the toggle or other important information.
+!!!
+
+!!! Dont
+
+Don’t add extra offset if the Rich Tooltip would block important information, appear disconnected from the element it is meant to provide information for, or cause confusion for the user.
+
+![Example of adjusting the offset of the Rich Tooltip](/assets/components/rich-tooltip/rich-tooltip-offset-dont.png)
+!!!
+
+## Interaction
+
+The Rich Tooltip supports two different interaction methods: a `hover/focus` ("soft") interaction and a `click` interaction. Which one to use depends on the type and complexity of content displayed by the component, whether _temporarily persisting_ the content in the UI is necessary or beneficial to the user, and whether the element toggling the Rich Tooltip is interactive or not.
+
+1. `Hover/focus` (default): displays when the mouse enters the toggle or the toggle receives focus. The `RichTooltip` has a timeout by default and will persist for 500 milliseconds (0.5 seconds) _after_ the mouse leaves the toggle, or the focus is removed from it.
+2. `On click`: displays when the user clicks the toggle with a mouse or if the toggle receives a keyboard event (`spacebar`, `enter/return`). If the content of the toggle element is interactive, an `on click` interaction _cannot_ be used.
+
+Independent of which interaction is used, the Rich Tooltip can be dismissed by clicking outside of the component or with the `ESC` key.
+
+## Toggle
+
+The Rich Tooltip supports both a default toggle (InfoText) to use the component in a consistent manner, but also supports generic or custom elements passed to the toggle element.
+
+### InfoText
+
+Be default, we provide the [InfoText](/components/rich-tooltip?tab=specifications#infotext-1) component which accounts for the majority of use-cases for the Rich Tooltip. This component ensures that the toggle is perceivable, visually consistent, and can be used inline with other content or standalone as part of the layout flow.
+
+!!! Info
+
+For now, the InfoText component should only be used with the Rich Tooltip. In Figma, this component is published separately for flexibility, while in Ember the component is coupled with the Rich Tooltip.
+!!!
+
+#### How to use
+
+Consider these guidelines when using the InfoText component:
+
+- Use the text + trailing icon variant by default.
```

Comment:
I believe I left a similar comment in Figma last week, but why are we providing text + icon variants, let alone recommending them as the default? This doubles up on the visual indication that the element is interactive, which adds to its weight and prominence in the UI. Why not recommend text only or icon only which should each provide enough visual indication on their own?

Reply:
I removed the default recommendation, but I think that providing flexibility for this element is the best solution because it supports the majority of use cases we've identified:

- Inline, as in a paragraph (text only)
- As a standalone or block element (icon leading or trailing)
- When paired with other UI elements that act as "labels" (icon only). This would be when using it with a badge or a button that aren't support as custom toggle elements.

Reply:
What is the default variant of the component (in Figma and code)?

Reply:
In code it's a "block". In Figma I don't think there is such differentiation.

@heatherlarsen did @jorytindall response above address the comment?